src/components/WebexMeetingGuestAuthentication/WebexMeetingGuestAuthentication.jsx

```diff
@@ -75,6 +97,15 @@ export default function WebexMeetingGuestAuthentication({
     adapter.meetingsAdapter.clearInvalidPasswordFlag(ID);
   };
 
+  const handleCaptchaChange = (value) => {
```

Comment:
Please utilize the useCallback hook  as well.

Reply:
Same as above